### ISSUE TITLE
Highlight matching do-end pairs

### ISSUE THREAD

**Issue by nitingupta910** [JakeBecker/vscode-elixir-ls#90]
For code like:
```
defmodule Greeter do
  def hello(name) do
    "Hello, " <> name
  end
end
```
currently, when one of `end` keywords is selected, the other `end` is highlighted. It would be great if matching do-end pairs are highlighted instead.

**Comment by JakeBecker** [comment]
This project has moved!

It's now being maintained by proactive volunteers from the Elixir community over at [elixir-lsp](https://github.com/elixir-lsp)/[vscode-elixir-ls](https://github.com/elixir-lsp/vscode-elixir-ls). The main ElixirLS codebase is now at [elixir-lsp](https://github.com/elixir-lsp)/[elixir-ls](https://github.com/elixir-lsp/elixir-ls). Updates will continue to be published from that repo to the original VS Code extension, so no need to switch plugins.

To avoid inundating the new maintainers with issues, please verify that your issue persists with the latest version of the extension (which is published from the new repo) before re-filing your issue there.

Thanks for using ElixirLS!

**Comment by lukaszsamson** [comment]
do-end (and fn-end) highlight matching landed in ElixirLS v0.19.0
